schedule/yast/autoyast_systemd_timesync.yaml

```diff
@@ -0,0 +1,28 @@
+name:       autoyast_systemd_timesync
+description:   >
+  Test performs autoyast installation where profile dictates that ntp is configured for one time synchronization.
+  Following, checks for the use of systemd timesync instead of cron, validates configuration and functionality.
+vars:
+  AUTOYAST: autoyast_sle15/autoyast_systemd_timesync.xml
+  AUTOYAST_PREPARE_PROFILE: 1
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/clone
+  - autoyast/verify_cloned_profile
+  - autoyast/logs
+  - autoyast/autoyast_reboot
+  - installation/grub_test
+  - installation/first_boot
+  - console/verify_systemd_timesync
+test_data:
+  profile:
+    ntp-client:
+      ntp_servers:
+        ntp_server:
+          address: cz.pool.ntp.org
```

Comment:
Hmm, we have this string written twice, so we have to change it in both places and one can forget about second place. What's about using variable expansion to set this value in the profile as well?

Reply:
Also, I don't see that we use it anywhere in validation, but I guess we should.

Reply:
I added a check for the server address, but about the variable expansion I have no idea how to do that.

Reply:
> One more thing not to forget is that we don't validate generated profile (clone installed system and validate ntp settings). Feel free to create follow up ticket for that, so we can merge this PR and proceed in the next iteration.

@rwx788 should I remove modules for cloning and verifying profile? what's the idea behind expanded variables? should I add some new variable for time sync server and expand it in autoyast profile? How is it going to be expanded in test_data? Should it be removed from there?

Reply:
@sofiasyria  You can check how is expanded {{SCC_REGCODE}} [here]
You could use this strategy and have some new setting variable (and document it) to use in both places, the xml profile and in the console validation check that you added recently to the test.

Currently for the validation of the profile we need to write it on more time, there is no expansion there.

Reply:
I added NTP_SERVER_ADDRESS as an expandable variable, replaced it in autoyast profile and I will open a new ticket for expanding variables in cloned profile. Did I get this right?

Reply:
yes, that's right, thanks!